src/thunderbots/software/ai/hl/stp/play/example_play.cpp

```diff
@@ -23,6 +25,15 @@ bool ExamplePlay::invariantHolds(const World &world) const
 std::vector<std::shared_ptr<Tactic>> ExamplePlay::getNextTactics(
     TacticCoroutine::push_type &yield, const World &world)
 {
+    AI::Passing::PassGenerator pass_generator(world, world.ball().position());
+
+    do {
+        pass_generator.setWorld(world);
+        pass_generator.setPasserPoint(world.ball().position());
+        std::this_thread::yield();
+        yield({});
+    } while(true);
```

Comment:
We should definitely remove this before merging, just have it here so people and pull at test out the changes.